nncf/common/graph/patterns/manager.py

```diff
@@ -143,5 +153,7 @@ def get_full_ignored_pattern_graph(
         :param model_type: ModelType instance.
         :return: Completed GraphPattern with registered value based on the backend, device & model_type.
         """
-        backend_patterns_map = PatternsManager._get_backend_ignored_patterns_map(backend)
+        backend_patterns_map = cast(
+            Dict[PatternNames, Callable[[], GraphPattern]], PatternsManager._get_backend_ignored_patterns_map(backend)
+        )
         return PatternsManager._get_full_pattern_graph(backend_patterns_map, device, model_type)
```

Comment:
In this function, the parameter `model_type` is optional. However, the function `_get_full_pattern_graph` expects `model_type` to be of type `ModelType` and not optional. The case where model_type is None is not explicitly handled. Is this a problem? (I didn't modify the datatype because I wanted to discuss with you this detail before, to figure out if I should modify `get_full_ignored_pattern_graph` or `_get_full_pattern_graph`)